### DISCUSSION TITLE
Help: I2C OLED SSD1306

### DISCUSSION THREAD

**Discussion by holefrog** [sle118/squeezelite-esp32#335] [Q&A] [answered]
Thank Renber78  for giving me the ideas, now I can connect an  I2C OLED SSD1306 to my A1s audio kit.

In the middle of the screen, it displays "SqueezeESP32" when booting, then displays "LMS IP:port".  But that is all. I hope it can display more information, e.g., tags of current playing song. 

My configs are:
"i2c_config":
SSD1306:
`sda=23,scl=5,port=1,speed=600000`

"display_config"
`I2C,driver=SSD1306,width=128,height=64`

**Comment by Renber78** [comment, marked as answer]
you simply need to install the SqueezeESP32 plugin on your LMS

**Reply by holefrog** [reply]
I am an LMS beginner, thank you so much. Now I have the tags. 
And the plugin is amazing, it converts Chinese characters into Pinyin. Do I have option to display Chinese character?

**Reply by Renber78** [reply]
No idea
